zipline/gens/brokers/robinhood_broker.py

```diff
@@ -0,0 +1,546 @@
+
+    @property
+    def positions(self):
+        z_positions = zp.Positions()
+        self._rh.update_info()
+        for symbol in self._rh.positions:
+            robinhood_position = self._rh.portfolio.positions[symbol]
+            try:
+                z_position = zp.Position(symbol)
+            except SymbolNotFound:
+                # The symbol might not have been ingested to the db therefore
+                # it needs to be skipped.
+                continue
+
+            z_position.amount = robinhood_position.amount
+            z_position.cost_basis = robinhood_position.cost_basis
+            z_position.last_sale_price = robinhood_position.last_price
+            z_position.last_sale_date = None 
+            z_positions[symbol] = z_position
+
+        return z_positions
+
+    @property
+    def portfolio(self):
+        z_portfolio = zp.Portfolio()
+        self._rh.update_info()
+        z_portfolio.capital_used = self._rh.portfolio.capital_used
+        z_portfolio.starting_cash = self._rh.portfolio.starting_cash 
+        z_portfolio.portfolio_value = self._rh.portfolio.portfolio_value
+        z_portfolio.pnl = self._rh.portfolio.pnl
+        z_portfolio.returns = self._rh.portfolio.returns
+        z_portfolio.cash = self._rh.portfolio.cash
+        z_portfolio.start_date = self._rh.portfolio.start_date
+        z_portfolio.positions = self._rh.portfolio.positions
+        z_portfolio.positions_value = self._rh.portfolio.positions_value
+        z_portfolio.positions_exposure = None
+
+        return z_portfolio
+
+    @property
+    def account(self):
+        z_account = zp.Account()
+        self._rh.update_info()
+        z_account.settled_cash = self._rh.account.settled_cash
+        z_account.accrued_interest = None
+        z_account.buying_power = self._rh.account.buying_power
+        z_account.equity_with_loan = self._rh.account.equity_with_loan
+        z_account.total_positions_value = self._rh.account.total_positions_value
+        z_account.total_positions_exposure = None
+        z_account.regt_equity = None
+        z_account.regt_margin = None
+        z_account.initial_margin_requirement = self._rh.account.initial_margin_requirement
+        z_account.maintenance_margin_requirement = self._rh.account.maintenance_margin_requirement
+        z_account.available_funds = self._rh.account.available_funds
+        z_account.excess_liquidity = self._rh.account.excess_liquidity
+        z_account.cushion = self._rh.account.cushion
+        z_account.day_trades_remaining = self._rh.account.day_trades_remaining
+        z_account.leverage = self._rh.account.leverage
+        z_account.net_leverage = self._rh.account.net_leverage 
+        z_account.net_liquidation = self._rh.account.net_liquidation
+
+        return z_account
+
+    @property
+    def time_skew(self):
+        return timedelta(seconds=0)
+        # return self._rh.time_skew
+
+    def order(self, asset, amount, limit_price, stop_price, style):
```

Comment:
This class uses the old API. We recently moved to use Blotter-Live, which changes some of the broker's call signatures. Could you please adopt your implementation too?